### PR TITLE
Add voicemails greetings API

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: wazo_calld_client/commands/voicemails.py [hunk 1]
```diff
@@ -57,18 +73,76 @@ def _move_message(self, url, dest_folder_id):
...
+            url, headers={'Content-type': 'audio/wav'}, data=data,
+        )
+        self.raise_from_response(r)
+
     def _get_recording(self, url):
         r = self.session.get(url, headers={'Accept': 'audio/wav'})
-        if r.status_code != 200:
-            self.raise_from_response(r)
-
+        self.raise_from_response(r)
```
Comment (fblackburn1) [inline wazo_calld_client/commands/voicemails.py]: Why do you remove the `if`? Have you seen that the `raise_from_response` is overridden by `CalldCommand` and didn't check the status code before raise? (agreed it is ugly)

Comment (sileht) [reply wazo_calld_client/commands/voicemails.py]: /me just fall in a well hidden trap :fearful:

Comment (sileht) [reply wazo_calld_client/commands/voicemails.py]: Oh it's worse that I thought, voicemails does not use CalldCommand :astonished:

Comment (sileht) [reply wazo_calld_client/commands/voicemails.py]: I will propose a separate PR to clean this mess :smile:

Comment (sileht) [reply wazo_calld_client/commands/voicemails.py]: I have also rebased on top of zuul/black PR

**Comment by sileht** [comment]
zuul ?

**Pull request merged**